Source/Core/InputCommon/ControllerEmu/ControlGroup/Tilt.h

```diff
@@ -20,7 +20,11 @@ class Tilt : public ReshapableInput
   explicit Tilt(const std::string& name);
 
   ReshapeData GetReshapableState(bool adjusted) final override;
-  ControlState GetGateRadiusAtAngle(double ang) const override;
+  ControlState GetGateRadiusAtAngle(double ang) const final override;
```

Comment:
You forgot the ones up here in the header ;)

Reply:
:+1: